components/mpas-ocean/cime_config/buildnml

```diff
@@ -1229,9 +1229,9 @@ def buildnml(case, caseroot, compname):
             lines.append('        filename_interval="output_interval"')
             lines.append('        clobber_mode="truncate"')
             lines.append('        packages="conservationCheckAMPKG"')
-            lines.append('        input_interval="none"')
+            lines.append('        input_interval="initial_only"')
             lines.append('        reference_time="01-01-01_00:00:00"')
-            lines.append('        output_interval="none">')
+            lines.append('        output_interval="stream:restart:output_interval">')
```

Comment:
Let's revert these changes.  I didn't understand that restarts are not needed because we are purposefully starting the average over at every output interval (i.e. accumulating only for 1 day).

Instead, let's change to `clobber_mode="append"` above in `conservationCheckOutput`.